documentation/GOVERNANCE.md

```diff
@@ -61,3 +61,16 @@ Modeling_Team
 to the PR. We have filters, which check for such comments. 
 ## Retiring a model
 Once a model gets the status "DEPRECATED" it should not be used anymore by new use cases. Existing applications should look for a way to migrate to an alternative version or model. In alignment with the overall release cycles of Tractus-X, the modeling team tags the current main branch with the corresponding Tractus-X release and publishes modeling release notes. These release notes contain a hint about all model versions that changed the status to "DEPRECATED" since the previous Tractus-X release. The modeling team will delete these models two Tractus-X releases cycles later. So use cases are advised to react to the deprecation of models within the next release cycle.
+
+## Versioning
+The semantic modeling in Tractus-X follows Semantic Versioning. Since the version is included in the Aspect Model's URI
+(i.e. `urn:bamm:io.catenax.batch:1.0.2#Batch`), applications that work on the model may have to adjust even if the APIs
+defined by the semantic model may not themselves change. The same phenomenon may occur with the CX-standards that reference
+a semantic model. Therefore, the following best practices shall be followed here:
+
+- No increment: Trivial changes that do not affect the functional parts of the model may be changed via PR without an incremented version number. 
+Changes must not affect the functional generated json-schema. This particularly applies to copyright-headers and descriptions.
+- Bugfix: A bugfix is a change in the model that does not add or remove functionality in the model. It is purely syntactical.
+Since it will however affect the APIs, the apps and standards must be adjusted accordingly.
+- Minor: A minor increment is widening the scope of a model by adding or removing features.
```

Comment:
The description of the model changes does not sound like what we agreed upon according to semantic versioning https://semver.org/lang/de/

We should adhere to the usual understanding of semantic versioning